src/odoc/page-index.mld

```diff
@@ -0,0 +1,252 @@
+{1:top odoc}
+
+The documentation compiler for OCaml, Reason, and BuckleScript.
+
++ {{:#overview} What is [odoc] really? }
++ {{:#flow} How does it work? }
++ {{:#deep-dive} Digging Deeper }
+
+{2:overview What is [odoc] really?}
+
+[odoc] is not your regular documentation tool.
+
+Instead of source code, it works with {i compilation units}; that is, it works
+on compiler outputs and turns them into compiled documentation, which then
+becomes gorgeous HTML.
+
+A regular [odoc] execution transforms [cmt], [cmti], and [mld] files into [odoc]
+files, and then turns those into [html] files. Roughly like this:
+
+{[
+open List
+
+[
+  Game.cmti;
+  Player.cmti;
+  Monster.cmti;
+  page-intro.mld;
+]
+|> map ~f:compile_to_odoc  (* here we have Game.odoc, Player.odoc, etc *)
+|> iter ~f:compile_to_html (* here we have Game/index.html, and so on  *)
+
+]}
```

Comment:
Wouldn't it be better to show this in shell syntax? Otherwise people will assume there's actually an OCaml API for this.